libraries/AP_Motors/AP_Motors_Class.h

```diff
@@ -122,6 +122,10 @@ class AP_Motors {
     // get motor interlock status.  true means motors run, false motors don't run
     bool                get_interlock() const { return _interlock; }
 
+    // get/set spoolup block
+    bool                get_spoolup_block() const { return _spoolup_block; }
+    void                set_spoolup_block(bool set) { _spoolup_block = set; }
```

Comment:
Should we only accept this if not already in throttle unlimited?

Reply:
@IamPete1 we could do that but it's redundant because set-desired-spool-state has the check there.  I'm not sure it's a good idea to have the check in both places.